tests/integration-tests/tests/dcv/test_dcv.py

```diff
@@ -44,11 +44,13 @@ def test_dcv_configuration(
     # dcv connect show url
     env = operating_system.environ.copy()
     env["AWS_DEFAULT_REGION"] = region
+    operating_system.system("sudo sed -i 's/\#   StrictHostKeyChecking ask/   StrictHostKeyChecking no/g' /etc/ssh/ssh_config")
     result = run_command(["pcluster", "dcv", "connect", cluster.name, "--show-url"], env=env)
     assert_that(result.stdout).matches(
         r"Please use the following one-time URL in your browser within 30 seconds:\n"
-        r"https:\/\/(\b(?:\d{1,3}\.){3}\d{1,3}\b):" + str(dcv_authenticator_port) + r"\?authToken=(.*)"
+        r"https:\/\/(\b(?:\d{1,3}\.){3}\d{1,3}\b):" + str(dcv_port) + r"\?authToken=(.*)"
     )
+    operating_system.system("sudo sed -i 's/\#   StrictHostKeyChecking ask/   StrictHostKeyChecking no/g' /etc/ssh/ssh_config")
```

Comment:
did you meant to revert the previous sed?

Reply:
Yes - fixed